docs/source/config/details.rst

```diff
@@ -228,3 +228,52 @@ a :ref:`startup file <startup_files>`::
 For more information on filters and what you can do with the ``event`` object,
 `see the prompt_toolkit docs
 <http://python-prompt-toolkit.readthedocs.io/en/latest/pages/building_prompts.html#adding-custom-key-bindings>`__.
+
+
+Enter to execute
+----------------
+
+As the exact behavior of :kbd:`Enter` is subject to disagreement, it has been
```

Comment:
I think we probably want to explain that we're talking about the heuristics of when to execute versus when to add a new line.